src/parsers/array.ts

```diff
@@ -1,37 +1,53 @@
-import { ZodTypeDef } from 'zod';
-import { ZodArrayDef } from 'zod/lib/src/types/array';
-import { JsonSchema7Type, parseDef } from '../parseDef';
+import { ZodArrayDef, ZodNonEmptyArray, ZodNonEmptyArrayDef, ZodTypeDef } from 'zod'
+// import { ZodArrayDef } from 'zod/lib/src/types/array';
+import { JsonSchema7Type, parseDef } from '../parseDef'
 
 export type JsonSchema7ArrayType = {
-  type: 'array';
-  items?: JsonSchema7Type;
-  minItems?: number;
-  maxItems?: number;
-};
+  type: 'array'
+  items?: JsonSchema7Type
+  minItems?: number
+  maxItems?: number
+}
 
-export function parseArrayDef(def: ZodArrayDef, path: string[], visited: { def: ZodTypeDef; path: string[] }[]) {
+export function parseArrayDef(def: ZodArrayDef | ZodNonEmptyArrayDef, path: string[], visited: { def: ZodTypeDef; path: string[] }[]) {
   {
     const res: JsonSchema7ArrayType = {
       type: 'array',
-      items: parseDef(def.type._def, [...path, 'items'], visited),
-    };
+      items: parseDef(def.type, [...path, 'array'], visited),
```

Comment:
I think the part of path should not be changed from 'items' to 'array'. At least by reverting it back to 'items' fixed a problem in my schema.

Reply:
changed to items https://github.com/StefanTerdell/zod-to-json-schema/pull/2/commits/211eb2ccf3d8c9acca6933ec4b45a86a94a84c3f